pkg/core/tekton/tekton.go

```diff
@@ -188,8 +201,39 @@ func (w *WorkflowBackend) CreateWorkflowListener(ctx context.Context, logger *lo
 		DashboardURL: fmt.Sprintf("%s/#/namespaces/%s/eventlisteners/%s", w.dashboardURL, w.namespace, el.Name)}, nil
 }
 
+// DeleteWorkflowListener deletes all tekton resources associated to the specified listener name
+func (w *WorkflowBackend) DeleteWorkflowListener(ctx context.Context, logger *log.Logger, name string) error {
```

Comment:
Here it is interesting that everything has same name: listener, triggerbinding and trigger template ... that's easy for us, but shouldn't we actually add some suffixes/prefixes when creating these objects?

Reply:
What would be the use of suffixes/prefixes?

Reply:
Don't you think that the naming "test-listener", "test-binding", "test-ttrigertemplate" is little more readable than "test", "test", "test"?